app/src/org/commcare/android/view/GridEntityView.java

```diff
@@ -142,11 +142,18 @@ public GridEntityView(Context context, Detail detail, Entity entity, String[] se
 		
 		//setup all the various dimensions we need
 		Point size = new Point();
```

Comment:
Since this variable is only used within the if() clause, should move its declaration there for clarity.